Python/Blood Alcohol Content/Blood_Alcohol_Content.py

```diff
@@ -0,0 +1,36 @@
+class BAC(object):
+    def __init__(self,weight, time, vol, amount, gender):
+        self.weight = weight
+        self.time = time 
+        self.vol = vol
+        self.amount = amount
+        self.gender = gender 
+        self.std = 0.0068 
+    def standard_drink(self):
+        return round((self.std * self.vol) * self.amount, 2)
+    #BAC formula for man 
```

Comment:
Nitpicking here but `# BAC formula for man`

Change man to males as well.
Update at other places accordingly.